lib/bloc/transactions/value_transfer/vtt_status/vtt_status_bloc.dart

```diff
@@ -33,15 +31,14 @@ class BlocStatusVtt extends Bloc<VttStatusEvent, VttStatusState> {
   static VttStatusState get initialState => UnknownHashState();
 
   Future<void> checkStatus(String transactionHash) async {
-    ApiExplorer apiExplorer = Locator.instance.get<ApiExplorer>();
+    // ApiExplorer apiExplorer = Locator.instance.get<ApiExplorer>();
```

Comment:
Are we calling checkStatus? If not, could we remove the method or add a comment explaining why we should keep it commented?